InfiniTAM/InfiniTAM_node.cpp

```diff
@@ -169,24 +184,24 @@ int main(int argc, char** argv) try {
   }
 
   printf("initialising ...\n");
-  ImageSourceEngine* image_source = NULL;
-  IMUSourceEngine* imu_source = NULL;
 
-  CreateDefaultImageSource(arg1, arg2, arg3, arg4, node_handle, image_source,
-                           imu_source);
+  SetUpSources(arg1, arg2, arg3, arg4, node_handle, image_source, imu_source);
```

Comment:
Eventually we can make it better in the new ros node in the branch external_tracker